src/Widgets/IconGroupContainer.vala

```diff
@@ -1,175 +1,162 @@
-//
-//  Copyright (C) 2014 Tom Beckmann
-//
-//  This program is free software: you can redistribute it and/or modify
-//  it under the terms of the GNU General Public License as published by
-//  the Free Software Foundation, either version 3 of the License, or
-//  (at your option) any later version.
-//
-//  This program is distributed in the hope that it will be useful,
-//  but WITHOUT ANY WARRANTY; without even the implied warranty of
-//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-//  GNU General Public License for more details.
-//
-//  You should have received a copy of the GNU General Public License
-//  along with this program.  If not, see <http://www.gnu.org/licenses/>.
-//
-
-namespace Gala {
-    /**
-     * This class contains the icon groups at the bottom and will take
-     * care of displaying actors for inserting windows between the groups
-     * once implemented
-     */
-    public class IconGroupContainer : Clutter.Actor {
-        public const int SPACING = 48;
-        public const int GROUP_WIDTH = 64;
-
-        public signal void request_reposition (bool animate);
-
-        private float _scale_factor = 1.0f;
-        public float scale_factor {
-            get {
-                return _scale_factor;
-            }
-            set {
-                if (value != _scale_factor) {
-                    _scale_factor = value;
-                    reallocate ();
-                }
+/*
+ * Copyright 2014 Tom Beckmann
+ * Copyright 2025 elementary, Inc. <https://elementary.io>
```

Comment:
You should use SPDX-FileCopyrightText: here

Reply:
Updated it :)